deploy/K8s-onprem-MIG/Readme

```diff
@@ -0,0 +1,5 @@
+This repo provides the scripts for autoscaling Triton deployment with Kubernetes (on MIG) and load balacing using NGNIX Plus.
```

Comment:
Every file needs nvidia copyright. You can find examples in the other deploy/ subdirectories

Reply:
added to each file